zulipterminal/ui_tools/boxes.py

```diff
@@ -373,23 +373,31 @@ def main_view(self) -> List[Any]:
                      .days)),
             'timestamp': (message['last']['time'] is not None and
                           message['this']['time'] != message['last']['time']),
-            'star_status': (message['this']['is_starred'] !=
-                            message['last']['is_starred']),
         }
         any_differences = any(different.values())
 
+        content_header = 0
+        flag = 0 # content_header needed if == 1
```

Comment:
A more descriptive variable name for example `is_starred_content_header` or something which means the same would help in better readability here. 

(*Note: inline comments are generally followed by 2 spaces before #, else linting fails*)

Reply:
Updated. 
Thanks!